### DISCUSSION TITLE
Was it intended that leaving the sampler set to "default" would use no sampler? [SD 1.5 model]

### DISCUSSION THREAD

**Discussion by CalculonPrime** [vladmandic/sdnext#3574] [Q&A]
I noticed that with my favorite SD 1.5 model, Absolute Reality 1.8.1, I get clown faces unless I change the sampler from "default" to something else - "DPM++ 2M" works fine.

Was that the intent? Not the clown faces, but not using a sampler by default?

- Promp: "Natalie Wood", 512x512, No sampler (default)
 ![00122- SD 1 5  absolutereality_v181-Natalie Wood](https://github.com/user-attachments/assets/58101a5c-11a1-4a78-aeae-b5c3e06f3958)
- Prompt: "Natalie Wood", 515x512, DPM++ 2M 
![00121- SD 1 5  absolutereality_v181-Natalie Wood](https://github.com/user-attachments/assets/309b8bfd-b081-449e-8afe-82ec224e7a8b)

**Comment by vladmandic** [comment]
default means whatever is set by model author. which for sd15 models is typically old-schoold ddim. it doesn't mean no sampler as that would result in absolutely nothing. and you're not getting clown faces, you're getting under-developed image since those samplers need more steps to resolve.

**Reply by CalculonPrime** [reply]
I assumed no sampler was being used when I select "default" because no sampler is listed in the image metadata.  For example:
"Parameters: Steps: 20| Seed: 3980765631| CFG scale: 6| Size: 512x512| Parser: Full| Model: [SD 1.5] absolutereality_v181| Model hash: 463d6a9fe8| Backend: Diffusers| App: SD.Next| Version: unknown| Operations: txt2img| Pipeline: StableDiffusionPipeline"

When I set DDIM explicitly, the "clown face" problem as I call it goes way, so that must not be what the model is using as a built-in default.  Is it possible there's no default sampler?

This is likely one of those areas where you could make the default experience better by auto-selecting something reasonable using heuristics.  Yes, I if the user has a high knowledge level, perhaps this isn't needed, but it would impress new users of your web UI.

**Reply by vladmandic** [reply]
no sampler would mean no steps at all and instant blank image. if it's running even a single step, it's running a sampler.

**Reply by Disty0** [reply]
Default sampler set by the original sd 1.5 model config is PNDM.
https://huggingface.co/stable-diffusion-v1-5/stable-diffusion-v1-5/blob/main/scheduler/scheduler_config.json
